index.html

```diff
@@ -44,8 +48,169 @@ <h1 class="prmrytxt1">
     <li><a href="#"><img src="./images/cptM.png" alt="cptM"></a></li>
     <li><a href="#"><img src="./images/hand.png" alt="Hand"></a></li>
   </ul>
-</section>
-  </main>
+  </section>
+
+  <!--Portfolio Section-->
+  <section class="works">
+    <div class="port1">
+      <img src="images/card1pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Tonic</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div >
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
```

Comment:
[OPTIONAL] I can see you have put in a lot on your project so far, keep it up😉 I think your project's card is slightly different from the given design. apart from it being a requirement that you stick to the design, being able to stick to the given design is a quality a recruiter will admire😉

your app currently.
<img width="268" alt="yourAPPpROJECT" src="https://user-images.githubusercontent.com/64643995/212461476-aa1038eb-ee1d-4adf-a984-bedd225dba9f.png">

the Figma design
<img width="266" alt="figma-your-App" src="https://user-images.githubusercontent.com/64643995/212461485-9386f41f-caee-4872-a2a4-7e43b5ce5f31.png">

Kindly consider the spacing, text color, font-properties and the button and kindly  make the changes🤗